File: Selenium/Activities/Activity6.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.firefox.service import Service as fireFoxService
from selenium.webdriver.chrome.service import Service as chrome
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.common.keys import Keys


service=fireFoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service)as driver:
    driver.get("https://training-support.net/selenium/input-events")    
    print("Page title:- " +driver.title)

    actions=ActionChains(driver)
    actions.send_keys(Keys.SHIFT+'s')
    actions.key_down(Keys.CONTROL).send_keys("a").send_keys("C").key_up(Keys.CONTROL).perform()

    

     

File: Selenium/Activities/Activity6.java
package Activity;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity6 {
    public static void main(String[] args) throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/selenium/input-events");
        System.out.println("Page Title:- " + driver.getTitle());

        Actions act=new Actions(driver);
        act.sendKeys(Keys.SHIFT).sendKeys("s").perform();

//        act.sendKeys(Keys.CONTROL+"a").sendKeys(Keys.CONTROL+"c").build().perform();
        act.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).build().perform();
        System.out.println(driver.findElement(By.id("keyPressed")).getText());

        driver.close();


    }
}
